File: Data Structures & Algorithms/non-cyclical-number/submission-0.cpp
class Solution {
public:
    bool isHappy(int n) {
        int sum=0;
        if(n==1)return true;
        while(n>1 && n!=4){
            sum=0;
            while(n>0){
                sum=sum+((n%10)*(n%10));
                n=n/10;
            }
            n=sum;
        }
        return n==1;
    }
};

File: Data Structures & Algorithms/find-words-that-can-be-formed-by-characters/submission-0.py
class Solution:
    def countCharacters(self, words: List[str], chars: str) -> int:
        sum=0
        for word in words:
            s=True
            for ch in word:
                if ch not in chars:
                    s=False
                    break
            if s==True:
                sum+=len(word)
        return sum
        